liberty-ls/src/main/java/io/openliberty/tools/langserver/utils/XmlReader.java

```diff
@@ -55,9 +56,11 @@ public static Map<String, String> getElementValues(File file, Set<String> elemen
 
         XMLInputFactory factory = XMLInputFactory.newInstance();
         try {
-            factory.setProperty(XMLInputFactory.IS_VALIDATING, Boolean.FALSE);
-            factory.setProperty(XMLInputFactory.IS_SUPPORTING_EXTERNAL_ENTITIES, Boolean.FALSE);
-            factory.setProperty(XMLInputFactory.SUPPORT_DTD, Boolean.FALSE);
+          factory.setProperty(XMLInputFactory.IS_VALIDATING, Boolean.FALSE);
```

Comment:
Same comment about updating the copyright year. Also, the formatting is off here. There should be an indent of 4 spaces not 2. That's why this PR shows changes to lines 59-61 even though the content is the same.